src/singlelayerqg.jl

```diff
@@ -218,7 +223,7 @@ addforcing!(N, sol, t, clock, vars::Vars, params, grid) = nothing
 function addforcing!(N, sol, t, clock, vars::ForcedVars, params, grid)
   params.calcF!(vars.Fh, sol, t, clock, vars, params, grid)
   @. N += vars.Fh
-  
+
```

Comment:
I think this is just Atom's `whitespace` package adding/removing spaces and make it as if you've modified more than you did... I had the same problem previously... @liasiegelman, perhaps you can disable `whitespace` package from Atom?